updatepermissionbuilder.go

```diff
@@ -24,30 +28,45 @@ func (upb *UpdatePermissionBuilder) UpdatePermission(address, channel *xdr.ID, n
 	return upb
 }
 
-func (upb *UpdatePermissionBuilder) Action(permissionAction int32) *UpdatePermissionBuilder {
-	upb.permissionAction = xdr.PermissionAction(permissionAction)
+func (upb *UpdatePermissionBuilder) Address(address xdr.ID) *UpdatePermissionBuilder {
+	upb.key = address
 	return upb
 }
 
-func (upb *UpdatePermissionBuilder) Address(address xdr.ID) *UpdatePermissionBuilder {
-	upb.key = address
+func (upb *UpdatePermissionBuilder) Alias(alias string) *UpdatePermissionBuilder {
+	upb.alias = alias
+	return upb
+}
+
+func (upb *UpdatePermissionBuilder) Authorize(accountUpdateType xdr.AccountUpdateType, account xdr.ID, alias string, authorize bool) *UpdatePermissionBuilder {
```

Comment:
I don't think AccountUpdateType should have to be set manually on this builder.  This should always use the `xdr.AccountUpdateTypeAUTHORIZATION` type.

Reply:
Done. Thanks.